disnake/channel.py

```diff
@@ -1009,6 +1040,31 @@ async def clone(
             {"bitrate": self.bitrate, "user_limit": self.user_limit}, name=name, reason=reason
         )
 
+    def is_nsfw(self) -> bool:
+        """:class:`bool`: Checks if the channel is NSFW."""
```

Comment:
Please use ``:return type: :class:`bool` `` as the current one would look like this:
![image](https://user-images.githubusercontent.com/67214928/143962065-5c410205-8862-4484-9cea-bf79db67b293.png)

note: I made a commit to change it in all docstrings [here](https://github.com/DisnakeDev/disnake/pull/153/commits/4457e3b90c6725df5f9c9d547e7acaa7b2798a91).